<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgResp</class>
 <widget class="QDialog" name="DlgResp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>363</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>234</red>
        <green>239</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>208</red>
        <green>219</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>234</red>
        <green>239</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>208</red>
        <green>219</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>208</red>
        <green>219</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>208</red>
        <green>219</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Persons in charge table</string>
  </property>
  <property name="windowIcon">
   <iconset resource="synoptux.qrc">
    <normaloff>:/images/Resp.png</normaloff>:/images/Resp.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTreeWidget" name="treeWidget_Resp">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">Code</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Label</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_code">
       <item>
        <widget class="QLabel" name="label_CodeBqe">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Code </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_CodeResp">
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="maxLength">
          <number>20</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_NomBqe">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Label</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_NomResp">
         <property name="maxLength">
          <number>30</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_coul_b">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Background color</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_CouleurFond">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_CouleurFondCode"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_ImageEuros">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>images/Euros.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="1" column="1" rowspan="2">
    <widget class="QPushButton" name="pushButton_ValidResp">
     <property name="text">
      <string>Validate</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" rowspan="2">
    <widget class="QPushButton" name="pushButton_annuler">
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" rowspan="2">
    <widget class="QPushButton" name="pushButton_QuitterResp">
     <property name="text">
      <string>Quit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_CodeResp</tabstop>
  <tabstop>pushButton_ValidResp</tabstop>
  <tabstop>pushButton_annuler</tabstop>
  <tabstop>pushButton_QuitterResp</tabstop>
 </tabstops>
 <resources>
  <include location="synoptux.qrc"/>
 </resources>
 <connections/>
</ui>
